src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -1259,6 +1259,14 @@ private static bool IsWord(SyntaxKind kind)
             return kind == SyntaxKind.IdentifierToken || IsKeyword(kind);
         }
 
+        private static bool IsWordOrLiteral(SyntaxKind kind)
+        {
+            return SyntaxFacts.IsLiteral(kind)
+                || IsKeyword(kind)
```

Comment:
>IsKeyword(kind) [](http://example.com/codeflow?start=19&length=15)

Why are we not reusing ```IsWord``` helper here? It looks like the check for an identifier is getting dropped. Is this intentional?

Reply:
Check for an identifier is part of `SyntaxFacts.IsLiteral`.